Sources/TuistKit/ProjectEditor/ProjectEditor.swift

```diff
@@ -120,4 +144,61 @@ final class ProjectEditor: ProjectEditing {
         try writer.write(workspace: descriptor)
         return descriptor.xcworkspacePath
     }
+
+    /// Attempts to load the plugins at the given path. If unable to find the Config manifest
+    /// displays a warning to the user.
+    /// - Returns: The loaded `Plugins`.
+    private func loadPlugins(at path: AbsolutePath) -> Plugins {
+        do {
+            let config = try configLoader.loadConfig(path: path)
+            return try pluginService.loadPlugins(using: config)
+        } catch {
+            logger.warning("Failed to load plugins, attempt to fix the Config manifest and rerun the command. Continuing...")
```

Comment:
I think here we should differentiate two scenarios:
- Plugins don't exist. In that case, we shouldn't print anything and just ignore them.
- The plugins fail to load. In this case, I'd expect the command to fail with a clear message on why it failed.

Reply:
@pepibumur 

> Plugins don't exist. In that case, we shouldn't print anything and just ignore them.

This is the behavior currently, so either the config doesn't exist, or it does. In either case `loadConfig` will return a default config or the actual config. From here we can get the plugins, if the are no plugins, this does not warn the user and we simply continue. 

> The plugins fail to load. In this case, I'd expect the command to fail with a clear message on why it failed.

Are we sure we want to fail here? My understanding is we should allow the usage of `tuist edit` under most contexts, even if the project may be broken (since they may be using `tuist edit` to fix said failure). For context, this warning would be thrown if we are unable to compile either the `Config.swift` or any of the `Plugin.swift` manifests.

Reply:
Fixed this up a bit, left a more detailed comment below